build.rs

```diff
@@ -1,5 +1,5 @@
 use anyhow::{Context as _, Error, Result};
-use git_repository as git;
+use gix as git;
```

Comment:
Using the `git` isn't necessary or recommended anymore - here it appears like a left-over though.
```suggestion
```

Instead, `gix` should be used directly which would require some modifications in `build.rs` as well.

Reply:
valid point, I renamed the usages